docker-compose.yaml

```diff
@@ -0,0 +1,248 @@
+  ldap:
+    image: osixia/openldap
+    ports:
+      - "389:389"
+      - "636:636"
+    volumes:
+      - ./ldif:/container/service/slapd/assets/config/bootstrap/ldif/custom
+    command: --copy-service
+    healthcheck:
+      test: ["CMD-SHELL", "ldapsearch -x -H ldap://localhost -b dc=example,dc=org -D 'cn=admin,dc=example,dc=org' -w admin"]
+      interval: 5s
+      timeout: 20s
+      retries: 3
+  zookeeper:
+    image: wurstmeister/zookeeper
+    ports:
+    - "2181:2181"
+  kafka:
+    image: wurstmeister/kafka
+    depends_on:
+    - zookeeper
+    ports:
+    - "9094:9094"
+    environment:
+    - "KAFKA_ADVERTISED_LISTENERS=INSIDE://:9092,OUTSIDE://${HOST_IP:-localhost}:9094"
+    - "KAFKA_LISTENER_SECURITY_PROTOCOL_MAP=INSIDE:PLAINTEXT,OUTSIDE:PLAINTEXT"
+    - "KAFKA_LISTENERS=INSIDE://:9092,OUTSIDE://:9094"
+    - "KAFKA_INTER_BROKER_LISTENER_NAME=INSIDE"
+    - "KAFKA_CREATE_TOPICS=streaming-transformed:1:1,streaming-dead-letters:1:1,streaming-persisted:1:1"
+    - "KAFKA_ZOOKEEPER_CONNECT=zookeeper:2181"
+    healthcheck:
+      test: ["CMD-SHELL", "cat /opt/kafka/logs/server.log | grep \"Previous Leader Epoch was: -1\""]
+      interval: 10s
+      timeout: 20s
+      retries: 3
+  zookeeper_public:
+    image: wurstmeister/zookeeper
+    # ports:
+    # - "3181:2181"
+  kafka_public:
+    image: wurstmeister/kafka
+    depends_on:
+    - zookeeper
+    # ports:
+    # - "9394:9094"
+    environment:
+    - "KAFKA_ADVERTISED_LISTENERS=INSIDE://:9092,OUTSIDE://${HOST_IP:-localhost}:9094"
+    - "KAFKA_LISTENER_SECURITY_PROTOCOL_MAP=INSIDE:PLAINTEXT,OUTSIDE:PLAINTEXT"
+    - "KAFKA_LISTENERS=INSIDE://:9092,OUTSIDE://:9094"
+    - "KAFKA_INTER_BROKER_LISTENER_NAME=INSIDE"
+    - "KAFKA_CREATE_TOPICS=streaming-transformed:1:1,streaming-dead-letters:1:1,streaming-persisted:1:1"
+    - "KAFKA_ZOOKEEPER_CONNECT=zookeeper_public:2181"
+    healthcheck:
+      test: ["CMD-SHELL", "cat /opt/kafka/logs/server.log | grep \"Previous Leader Epoch was: -1\""]
+      interval: 10s
+      timeout: 20s
+      retries: 3
+  metastore:
+    image: smartcitiesdata/metastore-testo:development
+    depends_on:
+    - postgres
+    ports:
+    - "9083:9083"
+    command: >
+      /bin/bash -c "
+        /opt/hive-metastore/bin/schematool -dbType postgres -validate || /opt/hive-metastore/bin/schematool -dbType postgres -initSchema;
+        /opt/hive-metastore/bin/start-metastore"
+  postgres:
+    logging:
+      driver: none
+    image: smartcitiesdata/postgres-testo:development
+    ports:
+    - "5432:5432"
+  minio:
+    image: smartcitiesdata/minio-testo:development
+    ports:
+    - "9000:9000"
+  presto:
+    depends_on:
+    - metastore
+    - minio
+    image: smartcitiesdata/presto-testo:development
+    ports:
+    - "8081:8080"
+    healthcheck:
+      test: ["CMD-SHELL", "curl -s http://localhost:8080/v1/info | grep -q '\"starting\":false'"]
+      interval: 10s
+      timeout: 30s
+      retries: 10
+  data-generator:
```

Comment:
Way to be resourceful on this! 
I'm going to request changes that we don't merge this in until we've played the open source card so this points at an actually built image.

Reply:
https://hub.docker.com/r/smartcitiesdata/data-generator
https://hub.docker.com/r/smartcitiesdata/data-generator
https://hub.docker.com/r/smartcitiesdata/data-generator